### PR TITLE
Update to 0.24

### PR DESCRIPTION
**Pull request by inducer**
Continued form #27.

cc @matthiasdiener

### CODE REVIEW

**Comment by inducer** [comment]
@conda-forge-admin please rerender

File: recipe/isl-strdup-win.patch [hunk 1]
```diff
@@ -0,0 +1,336 @@
+diff --git a/include/isl/ctx.h b/include/isl/ctx.h
+index ecd9272e..3fcba36f 100644
+--- a/include/isl/ctx.h
++++ b/include/isl/ctx.h
+@@ -233,7 +233,7 @@ isl_stat prefix ## _set_ ## field(isl_ctx *ctx, const char *val)	\
+ 	if (!val)							\
+ 		return isl_stat_error;					\
+ 	free(options->field);						\
+-	options->field = strdup(val);					\
++	options->field = __strdup(val);					\
```
Comment (matthiasdiener) [inline recipe/isl-strdup-win.patch]: There are two underscores here, is this intentional?

Comment (inducer) [reply recipe/isl-strdup-win.patch]: Thanks, good catch! 94b7af0

File: recipe/meta.yaml [hunk 2]
```diff
@@ -44,3 +46,4 @@ extra:
   recipe-maintainers:
     - frol
     - isuruf
+    - inducer
```
Comment (inducer) [inline recipe/meta.yaml:49]: FYI. View this as an offer, happy to revert.

Comment (frol) [reply recipe/meta.yaml:49]: I’m happy to pass it over! 🙏

**Comment by inducer** [comment]
I think this ready to go.

**Comment by matthiasdiener** [comment]
@conda-forge-admin, please rerender

**Comment by matthiasdiener** [comment]
This will need manual merge @isuruf.

**Comment by isuruf** [comment]
Thanks

**Pull request merged**